### PR TITLE
Update ko_KR.json

### PR DESCRIPTION
**Pull request by DDinghoya**

### CODE REVIEW

File: Ryujinx.Ava/Assets/Locales/ko_KR.json [hunk 1]
```diff
@@ -179,56 +180,56 @@
...
   "ControllerSettingsControllerTypeHandheld": "휴대용",
-  "ControllerSettingsControllerTypeProController": "Pro Controller",
-  "ControllerSettingsControllerTypeJoyConPair": "JoyCon 페어",
-  "ControllerSettingsControllerTypeJoyConLeft": "JoyCon 왼쪽",
-  "ControllerSettingsControllerTypeJoyConRight": "JoyCon 오른쪽",
-  "ControllerSettingsProfile": "프로필",
+  "ControllerSettingsControllerTypeProController": "프로 컨트롤러",
+  "ControllerSettingsControllerTypeJoyConPair": "죠이콘 페어링",
+  "ControllerSettingsControllerTypeJoyConLeft": "죠이콘 왼쪽",
+  "ControllerSettingsControllerTypeJoyConRight": "죠이콘 오른쪽",
```
Comment (KhoraLee) [inline Ryujinx.Ava/Assets/Locales/ko_KR.json:190]: `조이콘` is more commonly used translation.

Comment (DDinghoya) [reply Ryujinx.Ava/Assets/Locales/ko_KR.json]: Done! :)

**Review by DDinghoya — COMMENTED** [review]
Change from "죠이" to "조이"

File: Ryujinx.Ava/Assets/Locales/ko_KR.json [hunk 2]
```diff
@@ -185,9 +185,9 @@
   "ControllerSettingsControllerType": "컨트롤러 유형",
   "ControllerSettingsControllerTypeHandheld": "휴대용",
   "ControllerSettingsControllerTypeProController": "프로 컨트롤러",
-  "ControllerSettingsControllerTypeJoyConPair": "죠이콘 페어링",
-  "ControllerSettingsControllerTypeJoyConLeft": "죠이콘 왼쪽",
-  "ControllerSettingsControllerTypeJoyConRight": "죠이콘 오른쪽",
+  "ControllerSettingsControllerTypeJoyConPair": "조이콘 페어링",
+  "ControllerSettingsControllerTypeJoyConLeft": "조이콘 왼쪽",
+  "ControllerSettingsControllerTypeJoyConRight": "이콘 오른쪽",
```
Comment (KhoraLee) [inline Ryujinx.Ava/Assets/Locales/ko_KR.json:190]: Typo here

Comment (DDinghoya) [reply Ryujinx.Ava/Assets/Locales/ko_KR.json]: Thank you!

**Review by gdkchan — APPROVED** [review]

**Comment by AcK77** [comment]
As Ryujinx has migrated translation management to Crowdin, please see https://crwd.in/ryujinx if you would like to pursue this contribution. I close this PR.
Thanks.

**Comment by DDinghoya** [comment]
> As Ryujinx has migrated translation management to Crowdin, please see https://crwd.in/ryujinx if you would like to pursue this contribution. I close this PR. Thanks.

Done on Crowdin. Thank you!

**Pull request closed without merging**